src/OrderLinesList/OrderLinesList.js

```diff
@@ -59,13 +62,24 @@ const columnWidths = {
   funCodes: '18%',
 };
 
+const queryTemplate = `(
```

Comment:
`queryTemplate` is a very general name, could you please make a more specific name?

Reply:
Updated